### DISCUSSION TITLE
Caught plotting error: Overflowed park buffer, writing 10001 bytes. Space: 8864

### DISCUSSION THREAD

**Discussion by Sutejm** [Chia-Network/chia-blockchain#6545] [Support]
Hi, Could anyone help me understand what this means and whats caused this error?

Caught plotting error: Overflowed park buffer, writing 10001 bytes. Space: 8864

**Comment by loppefaaret** [comment]
It seems it tried to put more data into something that can't hold that much data ? do you have more to go on here ?

**Comment by BrandtH22** [comment]
@Sutejm , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
